### DISCUSSION TITLE
can't create a new repository

### DISCUSSION THREAD

**Discussion by tomabulash** [community/community#53912] [Repositories] [answered]
### Select Topic Area

Question

### Body

<img width="783" alt="image" src="https://user-images.githubusercontent.com/19751968/234971892-057d5886-dbe7-407c-9596-1577c94eed3e.png">

the 'checking availability...' message is showing continuously, and the 'create repository' button is non responsive,
please help...

**Comment by multikitty** [comment, marked as answer]
hi, @tomabulash , it can be delayed because of network issues in some cases.
Please check your network and refresh page.

**Reply by tomabulash** [reply]
thanks!

**Comment by oppahansi** [comment]
I have the same issue.

**Comment by thispaul** [comment]
Hi, this issue should now be resolved. Please let us know if you are still having trouble creating a repo.

**Reply by maurojfer** [reply]
> Hi, this issue should now be resolved. Please let us know if you are still having trouble creating a repo.

Hi, I'm still having trouble creating repositories, same as OP.

This is the request payload from Chrome Dev tools:

```
authenticity_token: undefined
value: configs
owner: companyname
json_response: true
```
EDIT: This error appears in the console

`157:1 Uncaught (in promise) SyntaxError: Unexpected token '<', "<!DOCTYPE "... is not valid JSON
a`

**Reply by th0Rn4Do** [reply]
> Hi, this issue should now be resolved. Please let us know if you are still having trouble creating a repo.

Hi @thispaul , I'm having a similar issue:
I tried creating a new repository.
The checking availability...' and '... is available' messages gets displayed as usual, but then I get an error saying: Something went wrong! OR 500 OR Unicorn.

**Reply by keyamam0** [reply]
@thispaul I've been getting the same issue of th0Rn4Do now ;(.

**Reply by keyamam0** [reply]
It worked via Firefox instead of Chrome using usually.

**Reply by th0Rn4Do** [reply]
> > Hi, this issue should now be resolved. Please let us know if you are still having trouble creating a repo.
> 
> Hi @thispaul , I'm having a similar issue: I tried creating a new repository. The checking availability...' and '... is available' messages gets displayed as usual, but then I get an error saying: Something went wrong! OR 500 OR Unicorn.

Solved.

**Comment by ingrsuarez** [comment]
Hi, I can't create a new repository. Something went wrong! message.

**Comment by PrashantBhatasana** [comment]
Hi @tomabulash 
As Github services are down, many users face the same issue.
you can check and confirm Github availability from https://www.githubstatus.com/incidents/f0mhbz9xn497 or follow https://twitter.com/githubstatus?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor
